taker-frontend/src/App.tsx

```diff
@@ -177,80 +177,78 @@ export const App = () => {
                 nextFundingEvent={nextFundingEvent}
                 referencePrice={referencePrice}
             />
-            <Box height={"100%"}>
-                <Center>
-                    <Box
-                        maxWidth={(VIEWPORT_WIDTH + 200) + "px"}
-                        width={"100%"}
-                        bgGradient={useColorModeValue(
-                            "linear(to-r, white 5%, gray.800, white 95%)",
-                            "linear(to-r, gray.800, white, gray.800)",
-                        )}
-                    >
-                        <Center>
-                            <Box
-                                textAlign="center"
-                                padding={3}
-                                bg={useColorModeValue(BG_LIGHT, BG_DARK)}
-                                maxWidth={VIEWPORT_WIDTH_PX}
-                                marginTop={`${HEADER_HEIGHT}px`}
-                                minHeight={`calc(100vh - ${FOOTER_HEIGHT}px - ${HEADER_HEIGHT}px)`}
-                                width={"100%"}
-                            >
-                                <Routes>
-                                    <Route path="/">
+            <Center>
+                <Box
```

Comment:
Can you replace the `Box` with `Center`?

Reply:
I played around with it for a bit but it's not straight forward - just moving the arguments to `Center` messes up the design. Am not investing more time at this point :)